README.md

```diff
@@ -69,18 +68,16 @@ To update dependencies, run
 To learn more about Cortex, consult the following documents / talks:
 
 - [Original design document for Project Frankenstein](http://goo.gl/prdUYV)
-- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
-- KubeCon Prometheus Day talk "Weave Cortex: Multi-tenant, horizontally scalable Prometheus as a Service" [slides](http://www.slideshare.net/weaveworks/weave-cortex-multitenant-horizontally-scalable-prometheus-as-a-service)
+- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/cortexproject/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
+- KubeCon Prometheus Day talk "Weave Cortex: Multi-tenant, horizontally scalable Prometheus as a Service" [slides](http://www.slideshare.net/cortexproject/weave-cortex-multitenant-horizontally-scalable-prometheus-as-a-service)
```

Comment:
These links should probably stay pointing at Weaveworks?

Reply:
fixed